test/mock/auth/in_memory_token_repository.py

```diff
@@ -5,18 +5,19 @@
 from typing import List
 
 from database.repositories import TokenRepositoryInterface
-from database.schemas import TokenTable, UserTable
+from database.schemas import TokenTable, UserTable, DriverProfileTable
 from database.exceptions import *
 
 
 class InMemoryTokenRepository(TokenRepositoryInterface):
-    def __init__(self, user_repo=None):
+    def __init__(self, user_repo=None, driver_repo=None):
```

Comment:
```suggestion
    def __init__(self, user_repository=None, driver_repository=None):
```
Pour que ca soit uniforme avec les autre InMemoryXxxXxxx classes